scripts/build.sh

```diff
@@ -43,21 +43,23 @@ if [ "${GOOS}x" == "windowsx" ]; then
 fi
 
 # Build needed plugins first
-ORIG_PATH=$(pwd);
-echo "==> Building Host Plugins..."
-for PLUGIN_TYPE in host; do
-    rm -f $ORIG_PATH/plugins/$PLUGIN_TYPE/assets/boundary-plugin-${PLUGIN_TYPE}*
-    for CURR_PLUGIN in $(ls $ORIG_PATH/plugins/$PLUGIN_TYPE/mains); do
-        cd $ORIG_PATH/plugins/$PLUGIN_TYPE/mains/$CURR_PLUGIN;
-        go build -v -o $ORIG_PATH/plugins/$PLUGIN_TYPE/assets/boundary-plugin-${PLUGIN_TYPE}-${CURR_PLUGIN}${BINARY_SUFFIX} .;
+if [ "${SKIP_PLUGIN_BUILD}x" == "x" ]; then
```

Comment:
Reviewers: This ensure we build both KMS and host plugins, and also allows for skipping plugin build if you know they're up-to-date for faster builds.